bazel/repository_locations.bzl

```diff
@@ -146,10 +146,10 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/jupp0r/prometheus-cpp/archive/81e208c250748657f1d5dab247e82c4429a931af.tar.gz"],
     ),
     com_github_libbpf_libbpf = dict(
-        sha256 = "3d6afde67682c909e341bf194678a8969f17628705af25f900d5f68bd299cb03",
-        strip_prefix = "libbpf-1.0.1",
+        sha256 = "4d503428c7aead070a59630dd0906318a430b3e279a35f51ec601fbdd7d31eb6",
+        strip_prefix = "libbpf-3b0973892891744d20ae79e99c0d1a26a59c4222",
         urls = [
-            "https://github.com/libbpf/libbpf/archive/refs/tags/v1.0.1.tar.gz",
+            "https://github.com/libbpf/libbpf/archive/3b0973892891744d20ae79e99c0d1a26a59c4222.tar.gz",
```

Comment:
This is the libbpf version that the 0.30.0 bcc release uses. It's a commit between [v1.1](https://github.com/libbpf/libbpf/releases/tag/v1.1.0) and [v1.2](https://github.com/libbpf/libbpf/releases/tag/v1.2.0)

![Screen Shot 2024-05-17 at 2 32 55 PM](https://github.com/pixie-io/pixie/assets/5855593/a150a344-b2d7-4819-a85a-98f5b6ef91c6)